test/component-bind-all-js.spec.js

```diff
@@ -689,15 +695,17 @@ describe('component', () => {
       expect(() => {
         widget.append(target);
       }).to.throw(Error,
-        'Binding "item.text" <-> "#foo.text" failed: Target property "text" does not perform type checks.'
+        'Binding "item.text" <-> "#foo.text" failed to initialize: Target property "text" '
+        + 'does not perform type checks.'
```

Comment:
Because the presence of a setter is just a plausibility check. We don't actually care whether there is a setter or not, we care that there is a type check and change events. A setter is needed for both. The most likely case here is that the user forgot to add a @property on the target, which generates the setter. Saying "there is no setter" could prompt the user to implement one by hand. But saying it is missing "@property" is also not correct since you CAN have a setter without it, like built-in widgets for example.

Reply:
Maybe append a suggestion in brackets? E.g. `(@property or setter missing?)`

Reply:
done